src/main/java/com/stripe/param/SubscriptionCancelParams.java

```diff
@@ -2,12 +2,18 @@
 
 import com.google.gson.annotations.SerializedName;
 import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import lombok.Getter;
 
 @Getter
 public class SubscriptionCancelParams extends ApiRequestParams {
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
```

Comment:
Do we know why `expand` is suddenly appearing here?

Reply:
A deliberate change by (I think) @rattrayalex-stripe in this commit: https://github.com/stripe/stripe-java/pull/922/commits/8028045a7147b957057f87aca742bdb76e6839b4.

Some DELETE endpoints return a full object (subscription cancel, source detach, maybe others) so this lets users expand fields on the object in those requests.